README.md

```diff
@@ -71,6 +71,12 @@ sudo update-alternatives --install /usr/bin/gcc gcc /usr/bin/gcc-9 100 --slave /
 sudo update-alternatives --config gcc
 ```
 
+The following environment variables can be set to influence the build or update of the environment.
+
+- ST_STKERNEL_TMPDIR (default: generated)
+- ST_STBOOT_URL (default: https://stboot.9esec.dev)
+- ST_QEMU_MEM (default: 8192)
+c
```

Comment:
Kernel tmpdir: 
This is now obsolete since there is no tmpdir anymore. Sources are downloaded to an untracked `src` dir inside the project folder. This comes in handy when looking up some kernle implementations / configurations during development.

Stboot URL: 
The SSH settings to upload the bootball to individual provisioning servers are configured in `stconfig/prov-server-access.sh` now. The URL in the generated `stboot/data/provisioning-servers.json` created by `stboot/data/create_example_data.sh` could be added to this config file as well maybe.

QEMU memory: 
I don't see the need to adjust this yet, since you need more memory to handle the downloaded bootball. But I'm not against making it configurable. Maybe it is time to introduce a global config file. There will be more things to go there I guess. QEMU memory and the settings from `stconfig/prov-server-access.sh` could go in such a file a beginning.

Reply:
I'll take a stab at a global config file. Simply sourcing such a file in scripts will make the various "sub-tasks", represented by their individual directories (deploy, stboot, stconfig), less independent. This could be solved by run.sh generating per-task config files, if we find that to be important later.

Reply:
I think it's OK if you cannot use the scripts inside the different directories on their own.
I've thought about it a few times yet, and always came to the conclusion that there is almost no need to run one of them stand alone.
Even if you need to call just one or twp scripts, it's always saver to call `run.sh` and skip everything but these two because this makes sure they are called in the right order.

Reply:
I think it depends on how much you want to automate the build. We should bring David in at some point.